includes/admin/class-sensei-setup-wizard.php

```diff
@@ -221,6 +220,8 @@ protected function redirect_to_setup_wizard() {
 	 * Render app container for setup wizard.
 	 */
 	public function render_wizard_page() {
+		// Delete option when the Setup Wizard is loaded, so it doesn't redirect anymore.
+		delete_option( 'sensei_activation_redirect' );
```

Comment:
Moved from the redirect method to here because we want it removed anytime the Setup Wizard is loaded.

I don't think it would happen in a normal flow, but this makes more sense in case some logic is changed in the future.